tests/functional/generate_pages.py

```diff
@@ -409,6 +427,19 @@ def process_final_summary(schema_data, page_spec, collapsible, section_summary=F
         page_spec.write(COLLAPSIBLE_SUMMARY_GETTER)
 
 
+def process_definition(definition_title, page_spec):
+    camel_case_definition_title = "".join(
+        x for x in definition_title.title() if not x.isspace()
+    )
```

Comment:
I think we need to consider using the index not the title. With a proper question, the ids and element names will be extremely long.

Reply:
I also have other content elements in there, the index will not be concurrent